src/gui/torrentoptionsdialog.ui

```diff
@@ -6,14 +6,71 @@
    <rect>
     <x>0</x>
     <y>0</y>
-    <width>390</width>
-    <height>450</height>
+    <width>450</width>
+    <height>540</height>
```

Comment:
please keep the old values.

Reply:
I added the "Download sequential" checkboxes which increased the minimum width and the same goes for height from the other options.

The other comments are addressed.